src/helpers/figures/charts/scorecard_chart.ts

```diff
@@ -458,6 +469,9 @@ export function createScorecardChartRuntime(
       text: chart.title.text ? _t(chart.title.text) : "",
     },
     keyValue: formattedKeyValue,
+    keyDescr: chart.keyDescr?.text
+      ? _t(chart.keyDescr.text) // descriptions are extracted from .json files and they are translated at runtime here
```

Comment:
the translation extraction script needs to be updated to take this into account then :wink: 

https://github.com/odoo/odoo/blob/93e53cb9628da05f5960a310ee1a4a311a689608/odoo/tools/translate.py#L1116